File: aplicacaomvc/Deloitte_NF.Web/Models/Fornecedor.cs

namespace Deloitte_NF.Models
{
    /// <summary>
    /// Classe Fornecedor
    /// </summary>
    public class Fornecedor
    {
        /// <summary>
        /// Obtém e informa o Sequencial do Fornecedor
        /// </summary>
        /// <value>Sequencial do Fornecedor</value>
        public int IdFornecedor { get; set; }

        /// <summary>
        /// Obtém e informa o Nome do Fornecedor
        /// </summary>
        /// <value>Nome do Fornecedor</value>
        public string NomeFornecedor { get; set; }

        /// <summary>
        /// Obtém e informa o CPF/CNPJ do Fornecedor
        /// </summary>
        /// <value>CPF/CNPJ do Fornecedor</value>
        public string CNPJFornecedor { get; set; }

        /// <summary>
        /// Obtém e informa o Endereço do Fornecedor
        /// </summary>
        /// <value>Endereço do Fornecedor</value>
        public string EnderecoFornecedor { get; set; }
    }
}

File: aplicacaowebforms/Negocio/Entidades/Fornecedor.vb
Public Class Fornecedor
    Inherits Generico

    Private _cnpjFornecedor As String
    Private __enderecoFornecedor As String

    Public Property CnpjFornecedor As String
        Get
            Return _cnpjFornecedor
        End Get
        Set(value As String)
            _cnpjFornecedor = value
        End Set
    End Property

    Public Property EnderecoFornecedor As String
        Get
            Return __enderecoFornecedor
        End Get
        Set(value As String)
            __enderecoFornecedor = value
        End Set
    End Property
End Class